1_basics.ipynb

```diff
@@ -183,18 +350,53 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Check that RViz has started on the simulator screen. In RViz, you can visualize various sensor information mounted on the robot.\n",
+    "\n",
+    "![title](./imgs/1_rviz.png)\n",
+    "\n",
+    "To switch between Gazebo and RViz, click [Gazebo] or [RViz] at the bottom of the screen."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "# スクリーンショット、スクリーンキャプチャの使い方"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# How to use screenshot and screencast"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "まず以下で必要なライブラリをインポートします。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "First, import the required libraries as below:"
```

Comment:
If we assume that you have a list of libraries, the sentence becomes:

"First, import the required libraries **listed** below:"